eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Gateway/DoctrineDatabase.php

```diff
@@ -101,6 +101,23 @@ public function setTable($name)
         $this->table = $name;
     }
 
+    /**
+     * Loads all list of aliases by given $locationId.
+     */
+    public function loadAllLocationEntries(int $locationId): array
+    {
+        $query = $this->connection->createQueryBuilder();
+        $query
+            ->select(...$this->columns[$this->table])
+            ->from($this->connection->quoteIdentifier($this->table))
+            ->where('action = :action')
+            ->andWhere('is_original= :is_original')
+            ->setParameter(':action', "eznode:{$locationId}", ParameterType::STRING)
```

Comment:
CS
```suggestion
            ->andWhere('is_original = :is_original')
            ->setParameter('action', "eznode:{$locationId}", ParameterType::STRING)
```

Reply:
Corrected, when it comes to formatting, I have restored the formatting before my changes to this file